qiskit/providers/basicaer/qasm_simulator.py

```diff
@@ -387,18 +385,11 @@ def _validate_measure_sampling(self, experiment):
             # measure sampling is allowed
             self._sample_measure = True
 
-    @deprecate_arg(
-        "qobj",
-        deprecation_description="Using a qobj for the first argument to QasmSimulatorPy.run()",
-        since="0.22.0",
-        pending=True,
-        predicate=lambda qobj: not isinstance(qobj, (QuantumCircuit, list)),
-    )
-    def run(self, qobj, **backend_options):
-        """Run qobj asynchronously.
+    def run(self, run_input, **backend_options):
+        """Run on the backend.
 
         Args:
-            qobj (Qobj): payload of the experiment
+            run_input (QuantumCircuit or Schedule or list): payload of the experiment
```

Comment:
A schedule isn't supported on BasicAer, it's just a circuit or a list of circuits:

```suggestion
            run_input (QuantumCircuit or list): payload of the experiment
```

Reply:
I wanted to express my gratitude for your approval! You're absolutely right; the lack of schedule support was an oversight on my part. I'll make sure to be more careful in my future PRs. Your feedback and attention to detail are highly valuable. Special thanks to @1ucian0 for updating the docstring.